### ISSUE TITLE
The only-one option is supported

### ISSUE THREAD

**Issue by laterdayi** [payam-zahedi/toastification#101]
The only-one option is supported

I want multiple clicks to show only the latest toast, just one

**Comment by payam-zahedi** [comment]
I guess you can close all the other toasts before showing the new one 

You can use this sample:

``` dart 
toastification.dismissAll();

toastification.show(...);

```

**Comment by payam-zahedi** [comment]
Is this what you need?

**Comment by laterdayi** [comment]
yes I think a lot of people need to hit it and only one toast comes out at the same time

**Comment by laterdayi** [comment]
Perhaps the official can provide an only-one option

**Comment by laterdayi** [comment]
It is expected that the official can provide the only-one option

**Comment by AntonioAEMartins** [comment]
Hi @laterdayi and @payam-zahedi,

I hope you're both doing well.

I was reading through the discussion on this issue and agree that a "one-only" toast option could be very useful, especially for mobile environments. If you're open to it, I'd love to contribute by implementing this feature. My suggestion is to introduce a only_one_mode variable in the ToastificationConfigProvider, which can be toggled between true or false to control this behavior. Let me know your thoughts, and if we align on this, I can start working on it.

Looking forward to your feedback!

**Comment by payam-zahedi** [comment]
Thats Right 

Will close the issue 

But feel free to reopen it if it makes sense for you
